op-node/rollup/driver/state.go

```diff
@@ -336,12 +348,35 @@ func (s *state) eventLoop() {
 				s.l2Head = unsafe
 				reqStep() // continue with the next step if we can
 			}
+		case respCh := <-s.syncStatusReq:
+			respCh <- SyncStatus{
+				CurrentL1:   s.derivation.Progress().Origin,
```

Comment:
this is the tip of what we may consider while working on, realistically going to be the l1Head. Mainly a comment on the fact that "current L1" is not well defined (but then fine to define as is).